### DISCUSSION TITLE
Fetching player data...

### DISCUSSION THREAD

**Discussion by aammgm** [cssbattle/forum#223] [Bugs reports]
Please help me
https://cssbattle.dev/player/peach

**Comment by Beowolve** [comment]
Works for me, ctrl+f5 maybe?

**Reply by aammgm** [reply]
I've tried.
![image](https://user-images.githubusercontent.com/49542674/153692949-d97f49e8-2b64-4699-a9a3-5c6f1d540257.png)

**Comment by chinchang** [comment]
We are checking in to this...please give us a few days to sort this out.

**Comment by rgnyldz** [comment]
I have the same issue. Not sure if related but my profile picture does not show up eighter.

**Comment by prokophanzl** [comment]
+1, same issue here.

**Comment by ghost** [comment]
I cannot see my score! 

![fetching_issue](https://user-images.githubusercontent.com/81631443/157286031-f43ec3a4-4607-448c-a723-168fd8a07472.jpg)
